src/migrations/1740007960791_AddAgendaItems.ts

```diff
@@ -0,0 +1,92 @@
+import { sql, type Kysely } from 'kysely';
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await sql`
+    CREATE TABLE "AgendaItem" (
+      "id" uuid PRIMARY KEY DEFAULT gen_random_uuid (),
+      "termId" INT NOT NULL,
+      "agendaItemId" INT NOT NULL,
+      "councilAgendaItemId" INT NOT NULL,
+      "decisionBodyId" INT NOT NULL,
+      "meetingId" INT NOT NULL,
+      "itemProcessId" INT NOT NULL,
+      "decisionBodyName" TEXT NOT NULL,
+      "meetingDate" BIGINT NOT NULL,
+      "reference" TEXT NOT NULL,
+      "termYear" TEXT NOT NULL,
+      "agendaCd" TEXT NOT NULL,
+      "meetingNumber" TEXT NOT NULL,
+      "itemStatus" TEXT NOT NULL,
+      "agendaItemTitle" TEXT NOT NULL,
+      "agendaItemSummary" TEXT NOT NULL,
+      "agendaItemRecommendation" TEXT,
+      "decisionRecommendations" TEXT,
+      "decisionAdvice" TEXT,
+      "subjectTerms" TEXT NOT NULL,
+      "wardId" json,
+      "backgroundAttachmentId" json,
+      "agendaItemAddress" json,
+      "address" json,
+      "geoLocation" json,
+      "planningApplicationNumber" TEXT,
```

Comment:
Any reason to prefer using `json` instead of `int[]` for the arrays of IDs? (it would give us easier access to things like any/all)

Reply:
norp!